runatlantis.io/docs/server-configuration.md

```diff
@@ -456,6 +456,21 @@ Values are chosen in this order:
   ```
   Log level. Defaults to `info`.
 
+### `--markdown-template-overrides-dir`
+  ```bash
+  atlantis server --markdown-template-overrides-dir="path/to/templates/"
+  ```
+  Directory where Atlantis will read in overrides for markdown templates used to render comments on pull requests.
+  Markdown template overrides may be specified either in individual files, or all together in a single file. All template
+  override files _must_ have the `.tmpl` extension, otherwise they will not be parsed.
+
+  Markdown templates which may have overrides can be found [here](../../server/events/templates)
```

Comment:
This link fails to resolve when the documentation is setup. Are you trying to link to the code base ? If so, you should use an absolute link to the github page instead of the relative link.

Reply:
@pseudomorph